### PR TITLE
[feature] add flush class

### PR DESCRIPTION
**Pull request by jun0315**

### CODE REVIEW

File: schema-engine-tag/src/main/java/org/apache/iotdb/lsm/request/IFlushRequest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+/** Represents a flush request that can be processed by the lsm framework */
+public interface IFlushRequest<K, V> extends IRequest<K, V> {
+
+  RequestType requestType = RequestType.FLUSH;
+
+  @Override
+  default RequestType getRequestType() {
+    return requestType;
+  }
```
Comment (MarcosZyk) [inline schema-engine-tag/src/main/java/org/apache/iotdb/lsm/request/IFlushRequest.java:29]: Can be simplified to "return RequestType.FlUSH"

Comment (KeeProMise) [reply schema-engine-tag/src/main/java/org/apache/iotdb/lsm/request/IFlushRequest.java:29]: IInsertionRequest,IDeletionRequest,IQueryRequest do the same.

Comment (jun0315) [reply schema-engine-tag/src/main/java/org/apache/iotdb/lsm/request/IFlushRequest.java]: Done

File: schema-engine-tag/src/main/java/org/apache/iotdb/lsm/context/requestcontext/FlushRequestContext.java [hunk 2]
```diff
@@ -27,7 +27,7 @@
 public class FlushRequestContext extends RequestContext {
   public FlushRequestContext() {
     super();
-    // use the reverse breadth-first traversal strategy to access memory nodes
-    accessStrategy = new RBFSAccessStrategy();
+    // use the reverse post-order traversal strategy to access memory nodes
```
[the remark this answers is not in the document]
Comment (jun0315) [reply schema-engine-tag/src/main/java/org/apache/iotdb/lsm/context/requestcontext/FlushRequestContext.java]: Done

**Pull request merged**